tools/flaky-test-reporter/jsonreport/jsonreport.go

```diff
@@ -60,37 +89,65 @@ func (r *Report) writeToArtifactsDir() error {
 	return ioutil.WriteFile(outFilePath, contents, 0644)
 }
 
+// getFlakyTests gets the latest flaky tests from the given repo
+func (c *JSONClient) GetFlakyTests(repo string) ([]string, error) {
+	return c.parseFlakyLog(repo, -1, func(report *Report, result *[]string) {
```

Comment:
```suggestion
	return c.parseFlakyLog(repo, -1, func(report *Report, result []string) {
```

Reply:
Ended up refactoring parseFlakyLog out, see the new changes